index.html

```diff
@@ -14,12 +14,55 @@
   <link rel="stylesheet" href="css/reset.css">
   <link rel="stylesheet" href="css/layout.css">
   <link rel="stylesheet" href="css/styles.css">
+  <link href="https://fonts.googleapis.com/css2?family=Open+Sans&family=Proza+Libre&display=swap" rel="stylesheet">
 </head>
 <body>
   <header>
+    <div class="containertwo">
+      <h3 class="heading-text">Code Journal</h3>
+    </div>
   </header>
   <main>
   </main>
+  <div data-view="entry-form" id="entry-form" class="entryformclass">
+    <form>
+      <div class="container">
+        <div class="column-full">
+          <div class="new-entry">
+            <p><b>New Entry</b></p>
```

Comment:
this should be a heading tag, you typically want to avoid using tags for CSS styling in this way.